script/gen_requirements_all.py

```diff
@@ -179,6 +180,9 @@
 # They are build with mypyc, but causes issues with our wheel builder.
 # In order to do so, we need to constrain the version.
 charset-normalizer==3.2.0
+
+# dacite: Ensure we have a version that is able to handle type unions.
```

Comment:
It might pay to mention the Roborock integration here, so people know where to look for regressions when changing this in the future

Reply:
I really thought I had done that. Thank you, I'll make the change when I'm on a desktop